app/models/event.py

```diff
@@ -229,6 +231,39 @@ def get_staff_roles(self):
         """returns only roles which are staff i.e. not attendee"""
         return [role for role in self.roles if role.role.name != ATTENDEE]
 
+    def get_tz_aware_time(self, time):
+        return pytz.timezone(self.timezone).localize(time)
+
+    @hybrid_property
+    def start_time_tz(self):
+        if not self.start_time:
+            return None
+        return self.get_tz_aware_time(self.start_time)
+
+    @hybrid_property
+    def end_time_tz(self):
+        if not self.end_time:
+            return None
+        return self.get_tz_aware_time(self.end_time)
+
+    @hybrid_property
+    def schedule_published_on_tz(self):
+        if not self.schedule_published_on:
+            return None
+        return self.get_tz_aware_time(self.schedule_published_on)
+
+    @start_time_tz.setter
```

Comment:
This error is simply because the Event model constructor (__init__) has no parameter named 'start_time_tz'.